zuul/tenants.yaml

```diff
@@ -78,6 +78,8 @@
               - ansible/molecule
               - ansible/workshops
               - metal3-io/metal3-dev-env
+              - pycontribs/pytest-molecule
+              - pycontribs/selinux
```

Comment:
Just to confirm, this will not allow us to gate the projects. We will be able to preform 3pci testing on the repo, but configuration will need to be loaded from here, project-config.

Reply:
Also, without the github app installed, we don't get events on new PRs, so this basically means depends-on only works.

Reply:
@pabelanger I installed Anzible Zuul app on these repos. Also I did sent you an invite there, just in case re-installation may be needed in the future.

Regarding effective job definitions, probably I will have to use the same approach I had on RDO in order to test them before running them on PRs.